gateway/v10.ts

```diff
@@ -1633,6 +1635,68 @@ export type GatewayUserUpdateDispatch = DataPayload<GatewayDispatchEvents.UserUp
  */
 export type GatewayUserUpdateDispatchData = APIUser;
 
+/**
+ * https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send
+ */
+export type GatewayVoiceChannelEffectSendDispatch = DataPayload<
+	GatewayDispatchEvents.VoiceChannelEffectSend,
+	GatewayVoiceChannelEffectSendDispatchData
+>;
+
+/**
+ * https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send
+ */
+export interface GatewayVoiceChannelEffectSendDispatchData {
+	/**
+	 * ID of the channel the effect was sent in
+	 */
+	channel_id: Snowflake;
+	/**
+	 * ID of the guild the effect was sent in
+	 */
+	guild_id: Snowflake;
+	/**
+	 * ID of the user who sent the effect
+	 */
+	user_id: Snowflake;
+	/**
+	 * The emoji sent, for emoji reaction and soundboard effects
+	 */
+	emoji?: APIEmoji | null;
+	/**
+	 * The type of emoji animation, for emoji reaction and soundboard effects
+	 * See https://discord.com/developers/docs/topics/gateway-events#voice-channel-effect-send-animation-types
+	 */
+	animation_type?: VoiceChannelEffectSendAnimationType;
+	/**
+	 * The ID of the emoji animation, for emoji reaction and soundboard effects
+	 */
+	animation_id?: number;
+	/**
+	 * The ID of the soundboard sound, for soundboard effects
+	 */
+	sound_id?: Snowflake;
+	/**
+	 * The filename of the soundboard default sound, for soundboard effects
+	 */
+	sound_override_path?: string | null;
```

Comment:
Not documented. Either mark as `@unstable` (if this is still included in the payload) or just get rid of it